src/auth/userSession.ts

```diff
@@ -100,15 +114,17 @@ export class UserSession {
   }
 
   /**
-   * Redirects the user to the Blockstack browser to approve the sign in request
-   * given.
+   * Redirects the user to the Blockstack browser to approve the sign in request. 
+   * To construct a request see the [[makeAuthRequest]] function.
    *
    * The user is redirected to the authenticator URL specified in the `AppConfig`
    * if the `blockstack:` protocol handler is not detected.
-   * Please note that the protocol handler detection
-   * does not work on all browsers.
-   * @param  {String} authRequest - the authentication request generated by `makeAuthRequest`
-   * @return {void}
+   * Please note that the protocol handler detection does not work on all browsers.
+   * 
+   * @param authRequest A request string built by the [[makeAuthRequest]] function
+   * @param blockstackIDHost The ID of the Blockstack Browser application.
+   * 
+   * @returns {MemoVoidIterator}
```

Comment:
Same here (looks like an accidental auto-import typo)

Reply:
Again...eslint.  Do you want me to turn off the linter?